emcc.py

```diff
@@ -1616,7 +1616,6 @@ def get_final():
         temp_basename = unsuffixed(final)
         wast_temp = temp_basename + '.wast'
```

Comment:
is this var no longer needed after the removal 2 lines below?

Reply:
Indeed, thanks